code/game/mecha/coral_gen.dm

```diff
@@ -0,0 +1,45 @@
+/obj/item/mecha_parts/mecha_equipment/coral_generator
+	name = "IA-C01G AORTA"
+	desc = "A highly classified emergent technology, burns raw redspace crystal to emit a red glow, greatly increasing the mech's movement speed, at the cost of greater energy usage."
```

Comment:
it specifically burns the dust TO glow red?

Reply:
it burns the crystal (in lore, no actual crystals are needed) to speed it up & make it schmove fast, I have to hit up molti to see if I can make the after-image only red

Reply:
I guess I should reword it to say it's more like an after-effect

Reply:
Description adjusted

Reply:
This still doesnt make sense. It doesnt burn dust to emit a red glow right?? That's just sorta an after effect of being evil as fuck right!?!?

Reply:
It burns the dust to enhance it's movement, a side effect of burning the dust is the after-image which is evil and red.